docs/CodeDevelopment/requirements.md

```diff
@@ -32,21 +32,50 @@
         }
     ```
 
-## Uefi Package
+## Repositories
```

Comment:
These lists are unordered where the other lists are ordered.  Should they be consistent?

Reply:
Fixed in latest.